app/src/main/java/io/github/droidkaigi/confsched2017/view/customview/OverScrollLayout.java

```diff
@@ -51,14 +49,16 @@ public OverScrollLayout(Context context, AttributeSet attrs, int defStyleAttr) {
     @Override
     protected void onLayout(boolean changed, int left, int top, int right, int bottom) {
         super.onLayout(changed, left, top, right, bottom);
-        originalLocation.set(left, top);
-        originalHeight = bottom - top;
-        overScrollThreshold = (int) (originalHeight * OVER_SCROLL_THRESHOLD);
+        originalRect.left = left;
```

Comment:
Rect also has `set` method.
[https://developer.android.com/reference/android/graphics/Rect.html#set(int, int, int, int)](https://developer.android.com/reference/android/graphics/Rect.html#set(int, int, int, int))

Can you use `set` instead?